cmd/duffle/pull.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/deislabs/duffle/pkg/duffle/home"
+	"github.com/deislabs/duffle/pkg/reference"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/cnab-to-oci/remotes"
+	"github.com/spf13/cobra"
+)
+
+type pullCmd struct {
+	output             string
+	targetRef          string
+	insecureRegistries []string
+	home               home.Home
+}
+
+func newPullCmd(w io.Writer) *cobra.Command {
+	var pull pullCmd
+	cmd := &cobra.Command{
+		Use:   "pull <ref> [options]",
+		Short: "Pulls an image reference",
```

Comment:
Does this command really pull an image reference?

Reply:
Yeah, thanks for catching this. Updating.